### DISCUSSION TITLE
`.env` export or copy to clipboard

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#1915] [Idea]
The new feature where you can paste your `.env` file directly into the Environment Variables section is super useful. It'd be amazing to be able to do the same thing in reverse? 

For example I'm setting up my project on a new machine, instead of copy pasting each Key Value pair individually etc. I could just click a `copy .env` button or something similar and paste it straight into my editor?

Thoughts?

**Comment by amyegan** [comment]
Hi, Mac! You can pull in your environment variables with the [`vercel pull`](https://vercel.com/docs/cli/pull#) CLI command. These will be used if you run the project locally with `vercel dev`. Or they can be copied from the `/.vercel` directory to wherever you need them

**Reply by ghost** [reply]
Yeah this is only useful for people that use the Vercel cli though. A web interface option would be useful for people that don't utilise the cli.
